app/views/about/hiring_community.html.haml

```diff
@@ -0,0 +1,153 @@
+= render ViewComponents::AboutNav.new(:hiring)
+
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header.c-shapes
+    .md-container
+      = link_to hiring_about_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-community", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Community Manager
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Community Manager!
+        We need someone to lead, grow and nurture our community of students, mentors and contributors.
+        If you're passionate about community, love people, and are excited about taking a systematic approach to creating a kind, supportive and helpful community, then this role might be perfect for you!
+
+      %h2.text-h2.mb-12 Summary
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free.
+        Our online platform is centred around self-directed learning and volunteer mentoring, and has nearly one million members.
+        Over the next 5 years, we want to become the defacto programming education platform that anyone, anywhere can use for free.
+        Through our work we aim to improve the global standard of programming and increase participation in tech from under-represented groups.
+        You can learn more about Exercism in our #{link_to '"About Exercism" pages', 'https://exercism.org/about', target: '_blank', rel: :noopener}.
+      %p.text-p-xlarge.mb-16
+        At its core, Exercism is a community. Our contributors have created thousands of exercises, our mentors have partipated in millions of discussions, and our students have helped and supported each other.
```

Comment:
Do we want one line per sentence?

```suggestion
        At its core, Exercism is a community.
        Our contributors have created thousands of exercises, our mentors have partipated in millions of discussions, and our students have helped and supported each other.
```

And maybe instead of "participated in" we can do:

```suggestion
        At its core, Exercism is a community.
        Our contributors have created thousands of exercises, our mentors have given feedback in millions of discussions, and our students have helped and supported each other.
```